### ISSUE TITLE
Comment random solution button?

### ISSUE THREAD

**Issue by erwo42** [exercism/exercism.io#3087]
Hi there!

I have never seen the link discussed in #988 or #734, so maybe I just missed it?

From the discussions there, I infer -- with some creative assumptions --, that that is not
what I want.

I would like a button "Comment random solution".

If I click this button:
1) Exercise I have done will be rolled first with some bias towards later exercises, to stop my boredom.
2) Some solution of someone else will be rolled, with bias towards solutions without comments and solutions, I have commented on are impossible to roll. There could be a bias towards more recently submitted solutions, just to rule out, that the submitter already lost interest and wouldn't look at my comment anyway. But that would be just a plus in my opinion.
3) I get presented that solution to comment on.

Such a button would be great or please point me to some similar feature if it already exists.

If I click "Dashbard", the link at "View Solutions", an then maybe the track button, the resulting
list doesn't change much and there seem to be ordered by time of last activity. That's not what
I want.

**Comment by kytrinyx** [comment]
> From the discussions there, I infer -- with some creative assumptions --, that that is not
> what I want.

:-) creative assumptions are welcome.

We had this behavior for a while, kind of, but it was hard to get right (lots of complicated edge cases that made the feature less useful than it should be), and we ended up taking it out in favor of a separate feature... which ended up being a performance problem that took down the database, so we removed that, too.

So, yeah :) I think we should have this feature, but we'll need to think about how to approach this in a way that doesn't put too much of a burden on the database.
